python/tank/pipelineconfig.py

```diff
@@ -858,31 +860,31 @@ def get_core_hooks_location(self):
         
         :returns: path string
         """
-        return os.path.join(self._pc_root, "config", "core", "hooks")
+        return os.path.join(self.get_config_location(), "core", "hooks")
```

Comment:
get_config_location() now returns a path straight into the bundle cache or inside a Toolkit classic pipeline.